packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue

```diff
@@ -155,8 +155,9 @@ export default defineComponent({
     const store = useStore()
 
     const linkListCollapsed = !store.getters.configuration.options.sidebar.shares.showAllOnLoad
-    const indirectLinkListCollapsed =
+    const indirectLinkListCollapsed = ref(
       !store.getters.configuration.options.sidebar.shares.showAllOnLoad
+    )
```

Comment:
Does this really work? Just wrapping a value in ref() shouldn't make it reactive I thought, doesnt this need to be wrapped in computed()?

Reply:
This works, yes. My idea was to make it mutable, because it needs to be written. `showAllOnLoad ` is just the initial value.

Reply:
Sounds legit @JammingBen we indeed don't want to mutate the config option in the store. using it as initial value and not persist the change in memory is a good solution. well spotted though @dschmidt I also had the same raised eyebrow feeling.

Reply:
Ah, one idea for improvement though:
```suggestion
const indirectLinkListCollapsed = ref(linkListeCollapsed)
```
Otherwise you have the same store reference twice in consecutive lines.